GUI/src/main/kotlin/ui/screens/SettingsScreen.kt

```diff
@@ -2,40 +2,55 @@ package ui.screens
 
 import androidx.compose.foundation.Image
 import androidx.compose.foundation.layout.*
-import androidx.compose.material.Button
+import androidx.compose.material.*
 import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.alpha
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.unit.dp
 import models.AppState
-import ui.components.CustomSlider
-import ui.components.FileSelectorButton
-import ui.components.textTitle
+import ui.components.*
 
 /**
  * SettingsScreen is the screen where the user can change the settings of the app.
  *
  * @param state the state of the app
  * @param oldState the previous state of the app
  */
-
 @Composable
 fun SettingsScreen(state: MutableState<AppState>) {
     val oldState = remember { mutableStateOf(state.value.copy()) }
+    val textForHyper =
+        "Settings to adjust behavior of the Gotoh alignment algorithm,\n" +
+            "which determines the matches between frames from both input videos."
+    val textForGapOpen =
+        "Penalty score for starting a new gap (insertion/deletion) in the alignment.\n" +
+            "Decreasing this value will favor fewer, larger gaps in the alignment."
+    val textForGapExtended =
+        "Penalty score for extending an existing gap (insertion/deletion) in the alignment.\n" +
+            "Increasing this value will favor longer gaps in the alignment.\n" +
+            "The gapOpenPenalty should be smaller than the gapExtensionPenalty.\n" +
+            "If both parameters are equal, the algorithm will behave like the Needleman-Wunsch algorithm,\n" +
+            "meaning that new gaps are as likely as extending existing gaps."
+    val textForMask =
+        "Upload a png with white and black rectangles.\n" +
+            "The area marked with white rectangles will be included in the video difference computation \n" +
+            "and the area with black rectangles will not be included in the computation."
```

Comment:
technically wrong:
Upload a png with a clear background and colored areas.\n" +
            "The clear areas will be included in the video difference computation \n" +
            "and the area with colored in will not be included in the computation